_maps/map_files/Pillar_of_Spring/TGS_Pillar_of_Spring.dmm

```diff
@@ -847,7 +847,7 @@
 /obj/machinery/reagentgrinder,
 /obj/structure/table/mainship,
 /obj/item/stack/sheet/mineral/phoron{
-	amount = 25;
+	amount = 5;
```

Comment:
Think you could use an object child with the amount you want here, instead of var-editing the amount?

Reply:
Theres no object child with a amount of 5, and I for one do not approve of creating 30 different child objects for material stacks

Reply:
You are creating several child objects when you var-edit in maps, but you are just leaving no trail on them.
This kind of thing makes refactors much harder.